### DISCUSSION TITLE
【第四期飞桨黑客松】模型套件开源贡献——目标检测开发套件

### DISCUSSION THREAD

**Discussion by cloud2009** [PaddlePaddle/Paddle#50725] [General]
[PaddlePaddle Hackathon 第四期](https://github.com/PaddlePaddle/Paddle/issues/50629#detection)
大家可以在这里讨论有关目标检测开发套件任务的问题哦～

**Comment by jjyaoao** [comment]
研发大佬您好，我想问问164号任务，提交的huggingface模型界面是不是类似于这样的？
![image](https://user-images.githubusercontent.com/88936287/221350713-b7f6a506-a166-4a27-bb6b-88cc699cc442.png)

另外PaddleDetection使用文档的提供，有没有参考的模板，以及文档应该提供的目录位置

**Reply by cloud2009** [reply]
@jerrywgz 您是这题的导师，请及时回复选手的提问，谢谢支持！

**Comment by jerrywgz** [comment]
嗯嗯模型界面是这样的，使用文档可以用这个链接 https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.6/deploy/pipeline/README_en.md

**Reply by jjyaoao** [reply]
> 嗯嗯模型界面是这样的，使用文档可以用这个链接 https://github.com/PaddlePaddle/PaddleDetection/blob/release/2.6/deploy/pipeline/README_en.md

研发老师，我是自己创建一个副标题为huggingface体验馆的区域，然后将模型和相关体验链接整理罗列上去，还是借用现在Readme里已有的某部分续写呢
